social_protection/workflows/base_beneficiary_update.py

```diff
@@ -98,9 +98,15 @@ def process_update_beneficiaries_workflow(user_uuid, benefit_plan_uuid, upload_u
             SET first_name = COALESCE(f."Json_ext"->>'first_name', first_name),
             last_name = COALESCE(f."Json_ext"->>'last_name', last_name),
             dob = COALESCE(to_date(f."Json_ext"->>'dob', 'YYYY-MM-DD'), dob),
+            location_id = loc."LocationId",
             "DateUpdated" = NOW(),
             "Json_ext" = f."Json_ext"
             FROM updated_beneficiaries f 
+            LEFT JOIN "tblLocations" AS loc
+                    ON loc."LocationName" = f."Json_ext"->>'location_name'
+                    AND loc."LocationCode" = f."Json_ext"->>'location_code'
```

Comment:
this is very constraining the code is enough

Reply:
Actually unciity in enforced https://github.com/openimis/openimis-be-location_py/blob/a41a615b888a81a49cd6dabdebeefa3ede35e416/location/services.py#L87